tests/wdio.conf.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * Ghostery Browser Extension
+ * https://www.ghostery.com/
+ *
+ * Copyright 2017-present Ghostery GmbH. All rights reserved.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0
+ */
+
+import path from 'node:path';
+import url from 'node:url';
+import { promises as fs } from 'node:fs';
+
+const __dirname = url.fileURLToPath(new URL('.', import.meta.url));
+const extPath = path.join(__dirname, '..', 'web-ext-artifacts');
+
+export let config = {
+  specs: ['onboarding.spec.js', 'privacy.spec.js'],
+  reporters: ['spec'],
+  logLevel: 'silent',
+  mochaOpts: { timeout: 60 * 1000 },
+};
+
+if (process.env.DEBUG) {
+  config = {
+    ...config,
+    specs: [config.specs],
+    capabilities: [
+      {
+        browserName: 'chrome',
+        'goog:chromeOptions': {
+          args: [`--load-extension=${path.join(__dirname, '..', 'dist')}`],
+        },
+      },
+    ],
+    logLevel: 'error',
+    mochaOpts: { timeout: 24 * 60 * 60 * 1000 },
+  };
+} else {
+  config = {
+    ...config,
+    specFileRetries: 5,
```

Comment:
would prefer to avoid that. if the tests are so flaky that we have many false negatives, then this setting may generate false positives

Reply:
Read the description carefully - this is for buggy web driver implementation in Firefox, this is not the issue of the tests itself.